### ISSUE TITLE
Fatal Error when 'head' block doesn't exist 

### ISSUE THREAD

**Issue by chedaroo** [leytech/Leytech_PrototypeUpdate#2]
In some case a fatal error is thrown if the `'head'` block doesn't exist (like some ajax responses)

```sh
Uncaught Error: Call to a member function getData() on boolean in ../.modman/leytech_leytech_prototypeupdate/src/app/code/community/Leytech/PrototypeUpdate/Model/Observer.php:26
```

**Comment by chedaroo** [comment]
#3 fixes

**Comment by bluec** [comment]
Hi @chedaroo thanks for the issue and PR. I have merged your PR and released a new version 1.0.1
